src/Workspaces/Core/Portable/Workspace/Host/DocumentService/ISpanMappingService.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Host
+{
+    /// <summary>
+    /// Map spans in a document to other spans even in other document
```

Comment:
can you clarify if you mean 'document' in an abstract sense, or 'Document' in the true "Roslyn Workspace Document" sense.

Reply:
yep. it is roslyn document

Reply:
can you update doc comment to use a cref?